main.py

```diff
@@ -55,12 +57,13 @@ def handle_regular_flow(jsonData):
         if processed is False:
             logger.error(f"Error processing the payload: {jsonData}")
             return jsonify(message="Something went wrong!"), 400
-        elif processed == -1:
+        if processed == -1:
             logger.warning("Contact not found in the payload")
             return jsonify(message="Contact"), 400
 
         if "contact" in jsonData:
             transaction_log_service.mark_webhook_log_as_processed(webhook_log)
+    return jsonify(message="Success"), 200
```

Comment:
We are sending success messages outside this function. So I don't thing we need this line

Reply:
Updated it to `none` as we are sending a success message from where this function is calling.